ARCs/arc-0069.md

```diff
@@ -97,7 +97,7 @@ However, contrary to the Asset URL, the `external_url` does not need to link to
 
 #### MIME Type
 
-In addition to specifying a data type in the ASA's URL (`au` field) with a URI fragment (ex: `#v` for video), the JSON Metadata schema also allows indication of the URL's MIME type (ex: `video/mp4`) via the `mime_type` field.
+In addition to specifying a data type in the ASA's URL (`au` field) with a URI fragment (ex: `#v` for video), the JSON Metadata schema also allows indication of the URL's MIME type (ex: `video/mp4`) via the `mime_type` field. For URL MIME 3D Mesh format **SHOULD** follow: https://www.iana.org/assignments/media-types/media-types.xhtml#model

```

Comment:
Sticking to meshes browsers can render effectively seems smart

Reply:
ARC69  as a standard is meant for Digital Media Tokens. It is not limited to browsers.  Using Iana standard can ensure  a wide application in  different devices, not only mobile devices, and a wider set of applicable internet standards and protocols.

Reply:
ARC69 has 69%+ marketshare because it works so well in browsers

Reply:
There is no marketshare problem. ARC 69 with these changes would continue to apply perfectly to everyone, giving the possibility of other types of uses.